genref/go.mod

```diff
@@ -3,31 +3,33 @@ module github.com/kubernetes-sigs/reference-docs/genref
 go 1.16
```

Comment:
Should you build with Go 1.17?

Reply:
Tried, but failed. Intra-package and inter-package references are not correctly resolved in 1.17. Need some time to investigate.